template/Template.csproj

```diff
@@ -1,8 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net7.0</TargetFramework>
         <PackageType>Template</PackageType>
-        <PackageVersion>6.1.1</PackageVersion>
```

Comment:
Why remove this?

Reply:
The version is now computed by MinVer like for the other packages.

Reply:
The version is now computed by MinVer, like for the Carter project/package.